<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">ChroniLaucnher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/images/imgs/chroniIcon.png</normaloff>:/images/imgs/chroniIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Images.qrc">:/images/imgs/chroniLogo.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>22</pointsize>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Welcome to Chroni! </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>79</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="appListWidget">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>317</width>
          <height>68</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autostartEnable">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Autostart (recommended)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionCreator_rubicpp"/>
    <addaction name="actionVersion_0_1_build"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionUkrainian"/>
    <addaction name="actionRussian"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionDark"/>
    <addaction name="actionLight"/>
    <addaction name="actionCustom"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuTheme"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light (TODO)</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom (TODO)</string>
   </property>
  </action>
  <action name="actionCreator_rubicpp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Creator: rubicpp</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionVersion_0_1_build">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Version: 0.1 build</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionUkrainian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ukrainian</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
